### DISCUSSION TITLE
Inquiry Regarding License Policy for Commercial Use of Security Onion

### DISCUSSION THREAD

**Discussion by 89sooner** [Security-Onion-Solutions/securityonion#13292] [Unsupported Versions]
Hello,

I am planning to install and set up Security Onion 2.4 for security monitoring. I intend to set up sensors in the network segments of affiliated companies within our group to detect events and perform monitoring, for which we will charge a service fee.

Additionally, we will provide a separate web interface developed based on Elasticsearch, but we will not offer major functionalities or access rights to Elasticsearch or Kibana.

I would like to inquire whether this constitutes commercial use under the license and if there are any issues with using Elastic License 2.0 (ELv2) in this context.

For reference, we will be using the ET Free version of Suricata rules.

Thank you.

**Comment by dougburks** [comment]
For inquiries regarding potential commercial use, please reach out to our commercial team by going to https://securityonionsolutions.com/support, clicking the Purchase Support button, filling out the contact form, and changing the description field as necessary. Thanks!
